CHANGELOG.rst

```diff
@@ -5,10 +5,17 @@ This document describes changes between each past release as well as
 the version control of each dependency.
 
 
-5.1.5 (unreleased)
+5.2.0 (unreleased)
 ==================
 
-- Nothing changed yet.
+kinto-amo
```

Comment:
Shall we also add the amo2kinto changelog here?

Reply:
Why not yes, it's more explicit.

To be honest I was surprised to see it as a direct dependency (and not internal stuff of kinto-amo)